### PR TITLE
Fix calc static balance point

### PR DESCRIPTION
**Pull request by Naoki-Hiraoka**
https://github.com/euslisp/jskeus/issues/540

:calc-static-balance-pointの引数について、

マニュアルでは
> force-list [N] and moment-list [Nm] are list of force and moment at target-points.

となっていますが、

現在の実装では
force-list: ロボットが受ける力 [N]
moment-list: ロボットが**与える**モーメント **[Nmm]**
となっています。

:calc-static-balance-pointの引数が
force-list: ロボットが受ける力 [N]
moment-list: ロボットが**受ける**モーメント **[Nm]**
となるように変更し、テストを追加しました。

### CODE REVIEW

File: irteus/test/test-irt-motion.l [hunk 1]
```diff
@@ -871,6 +871,48 @@
...
+                                    (send robot :calc-static-balance-point :force-list force-list :moment-list moment-list)))
+         (> 1.0
+            (norm (subseq (transform (send robot :calc-grasp-matrix
+                                           (append (mapcar #'(lambda (x) (send robot x :end-coords :worldpos)) (list :rarm :larm))
+                                                   (list (send robot :centroid)
+                                                         (send (send robot :foot-midcoords) :worldpos))))
+                                     (concatenate float-vector
+                                                  (mapcan #'(lambda (f m) (concatenate cons f m))
+                                                          force-list moment-list)
+                                                  (v- (scale (* 1e-6 (send *robot* :weight)) *g-vec*)) #F(0 0 0)
```
Comment (mmurooka) [inline irteus/test/test-irt-motion.l]: `*robot*` -> `robot`. Also the next line.

File: irteus/test/test-irt-motion.l [hunk 2]
```diff
@@ -871,6 +871,48 @@
...
+                :link-list (mapcar #'(lambda (x)
+                                       (send robot :link-list (send robot x :end-coords :parent)))
+                                   (list :rleg :lleg :rarm :larm))
+                :centroid-thre centroid-thre
+                :centroid-offset-func #'(lambda () (send robot :calc-static-balance-point :force-list force-list :moment-list moment-list))
+                :debug-view debug-view)))
+    (and result
+         (> centroid-thre (distance (apply #'midpoint 0.5 (send robot :legs :end-coords :worldpos))
+                                    (send robot :calc-static-balance-point :force-list force-list :moment-list moment-list)))
+         (> 1.0
```
Comment (mmurooka) [inline irteus/test/test-irt-motion.l]: Please add a comment about the meaning of this condition checking.

Comment (mmurooka) [reply irteus/test/test-irt-motion.l]: I follow this code, and now understand this condition check and think this is OK. (But still add comment please.)

**Comment by Naoki-Hiraoka** [comment]
Thank you for reviewing.
I fixed the document, added the comment, and fixed my code as suggested.

**Comment by mmurooka** [comment]
OK, and rebase into one commit and force push, please.

**Comment by Naoki-Hiraoka** [comment]
force pushed.

**Comment by mmurooka** [comment]
Thanks, approved.
Same fix with https://github.com/euslisp/EusLisp/pull/376 is necessary for travis test passing. (maybe another new PR should be created for this.)

**Comment by k-okada** [comment]
@Naoki-Hiraoka can we create documents with examples ? https://jskeus.readthedocs.io/ja/latest/irtmodel.html#inverse-kinematicstarget-coords

**Comment by Naoki-Hiraoka** [comment]
I tried adding documents in https://github.com/euslisp/jskeus/pull/549 . 

Should I update https://github.com/jsk-ros-pkg/euslisp-docs?

**Comment by k-okada** [comment]
> I tried adding documents in #549
> <https://github.com/euslisp/jskeus/pull/549> .
>

Thank you

> Should I update https://github.com/jsk-ros-pkg/euslisp-docs?
>

No you do not have to.
-- 
--
◉ Kei Okada

**Pull request merged**